examples/tools_support.ipynb

```diff
@@ -0,0 +1,7979 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "l68Urk97Wmut"
+      },
+      "source": [
+        "# 🛠️ Tools support\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/github/deepset-ai/haystack-experimental/blob/main/examples/tools_support.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\" width=\"200\"/></a>\n",
+        "\n",
+        "In this notebook, we introduce the experimental features we've been developing for tool/function calling support in Haystack.\n",
```

Comment:
Add something about what they will be building

Reply:
done